common/controllers/fps-controller.cpp

```diff
@@ -22,10 +22,10 @@ namespace tec {
 				Handle(key_event, state);
 			}
 		}
+		for (const MouseBtnEvent& mouse_button_event : commands.mouse_button_events) {
+			Handle(mouse_button_event, state);
+		}
```

Comment:
I moved this out of the mouse_focus during testing, it's probably okay until something ends up getting added that focuses the mouse, but isn't this whole class a stopgap anyways?